ui/raidboss/data/02-arr/dungeon/hullbreaker_isle.ts

```diff
@@ -0,0 +1,33 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.HullbreakerIsle,
+  triggers: [
+    {
+      id: 'Hullbreaker Isle Stool Pelt',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '89E', source: 'Sasquatch', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Hullbreaker Isle Chest Thump',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '89F', source: 'Sasquatch', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Give banana to Harambee',
```

Comment:
```suggestion
          en: 'Give banana to sasquatch',
```

Reply:
aww ... i thought ppl would understand that better tho =)

Reply:
It's better to be clear than funny, and definitely no racist memes, sorry.